server/genquery2/include/irods/private/genquery2_table_column_mappings.hpp

```diff
@@ -0,0 +1,233 @@
+        //{"QUOTA_MODIFY_TIME",       {"R_QUOTA_MAIN", "modify_ts"}},
+        //{"QUOTA_USAGE_USER_ID",     {"R_QUOTA_USAGE", "user_id"}},
+        //{"QUOTA_USAGE_RESC_ID",     {"R_QUOTA_USAGE", "resc_id"}},
+        //{"QUOTA_USAGE",             {"R_QUOTA_USAGE", "quota_usage"}},
+        //{"QUOTA_USAGE_MODIFY_TIME", {"R_QUOTA_USAGE", "modify_ts"}},
+        //{"QUOTA_USER_NAME",         {"R_USER_MAIN", "user_name"}}, // special?
+        //{"QUOTA_USER_TYPE",         {"R_USER_MAIN", "user_type_name"}}, // special?
+        //{"QUOTA_USER_ZONE",         {"R_USER_MAIN", "zone_name"}}, // special?
+        //{"QUOTA_RESC_NAME",         {"R_RESC_MAIN", "resc_name"}}, // special?
+
+        {"DATA_ACCESS_PERM_ID", {"R_OBJT_ACCESS", "access_type_id"}},
+        {"DATA_ACCESS_PERM_NAME", {"R_TOKN_MAIN", "token_name"}},
+        {"DATA_ACCESS_USER_ID", {"R_OBJT_ACCESS", "user_id"}},
+        {"DATA_ACCESS_USER_NAME", {"R_USER_MAIN", "user_name"}},
+        //{"DATA_ACCESS_DATA_ID",  {"R_OBJT_ACCESS", "object_id"}},
+        //{"DATA_ACCESS_NAME",     {"R_TOKN_MAIN", "token_name"}}, // special?
+        //{"DATA_TOKEN_NAMESPACE", {"R_TOKN_MAIN", "token_namespace"}}, // special?
+
+        {"COLL_ACCESS_PERM_ID", {"R_OBJT_ACCESS", "access_type_id"}},
+        {"COLL_ACCESS_PERM_NAME", {"R_TOKN_MAIN", "token_name"}},
+        {"COLL_ACCESS_USER_ID", {"R_OBJT_ACCESS", "user_id"}},
+        {"COLL_ACCESS_USER_NAME", {"R_USER_NAME", "user_name"}},
+        //{"COLL_ACCESS_COLL_ID",  {"R_OBJT_ACCESS", "object_id"}},
+        //{"COLL_ACCESS_NAME",     {"R_TOKN_MAIN", "token_name"}}, // special?
+        //{"COLL_TOKEN_NAMESPACE", {"R_TOKN_MAIN", "token_namespace"}}, // special?
+
+        // Ticket support is disabled until we figure out how to handle them.
+        // This protects against users gaining access to info they don't have access to.
+        //{"TICKET_ID", {"R_TICKET_MAIN", "ticket_id"}},
+        //{"TICKET_STRING", {"R_TICKET_MAIN", "ticket_string"}},
+        //{"TICKET_TYPE", {"R_TICKET_MAIN", "ticket_type"}},
+        //{"TICKET_USER_ID", {"R_TICKET_MAIN", "user_id"}},
+        //{"TICKET_OBJECT_ID", {"R_TICKET_MAIN", "object_id"}},
+        //{"TICKET_OBJECT_TYPE", {"R_TICKET_MAIN", "object_type"}},
+        //{"TICKET_USES_LIMIT", {"R_TICKET_MAIN", "uses_limit"}},
+        //{"TICKET_USES_COUNT", {"R_TICKET_MAIN", "uses_count"}},
+        //{"TICKET_WRITE_FILE_LIMIT", {"R_TICKET_MAIN", "write_file_limit"}},
+        //{"TICKET_WRITE_FILE_COUNT", {"R_TICKET_MAIN", "write_file_count"}},
+        //{"TICKET_WRITE_BYTE_LIMIT", {"R_TICKET_MAIN", "write_byte_limit"}},
+        //{"TICKET_WRITE_BYTE_COUNT", {"R_TICKET_MAIN", "write_byte_count"}},
+        //{"TICKET_EXPIRY_TIME", {"R_TICKET_MAIN", "ticket_expiry_ts"}},
+        //{"TICKET_CREATE_TIME", {"R_TICKET_MAIN", "create_time"}},
+        //{"TICKET_MODIFY_TIME", {"R_TICKET_MAIN", "modify_time"}},
+        ////{"TICKET_LOGICAL_PATH", {"R_TICKET_MAIN", "modify_time"}},
```

Comment:
lol - yep. That's on purpose and serves to help me when I start thinking about how to handle tickets.

I'll add a comment in case it turns out someone else picks up the torch for tickets.

Reply:
Done